auslib/test/admin/views/test_rules.py

```diff
@@ -307,7 +326,12 @@ def testPutRuleOutdatedData(self):
 
     def testPostRuleOutdatedData(self):
         # Make changes to a rule
-        ret = self._post('/rules/1', data=dict(backgroundRate=71, mapping='d', priority=73, data_version=1, product='Firefox', channel='nightly'))
+        ret = self._post('/rules/1', data=dict(backgroundRate=71,
+                                               mapping='d', priority=73,
+                                               data_version=1, product='Firefox', channel='nightly'))
+        self.assertEquals(ret.status_code, 400, "Status Code: %d, Data: %s" % (ret.status_code, ret.data))
+        ret = self._post('/rules/1', data=dict(backgroundRate=71, mapping='d', priority=73, data_version=1,
+                                               product='Firefox', channel='nightly', update_type='major'))
```

Comment:
No need to add an extra post() here - just add update_type to the existing one. Same goes for the other tests where you're doing this. Unless the test is explicitly testing the update_type case, there's no reason to check or verify it.